apps/rackstep.ru

```diff
@@ -0,0 +1,13 @@
+require 'rackstep'
+
+class HelloWorld < RackStep::App
+  add_route('GET', '', 'HelloWorldController')
```

Comment:
Will answer only to the root (/) path.

The /foo path will be get a 404 (not found).